[Bug] i.fusion.hpf can longer be installed with g.extension

**Name of the addon**

`i.fusion.hpf`

**Describe the bug**

I cannot install the addon any more (this worked some months ago):

```
g.extension i.fusion.hpf

Fetching <i.fusion.hpf> from <https://github.com/OSGeo/grass-addons/> (be
patient)...
remote: Enumerating objects: 27, done.
remote: Counting objects: 100% (10/10), done.
remote: Compressing objects: 100% (10/10), done.
remote: Total 27 (delta 0), reused 0 (delta 0), pack-reused 17
Receiving objects: 100% (27/27), 1.07 MiB | 11.31 MiB/s, done.
Already on 'grass8'
Your branch is up to date with 'origin/grass8'.
Compiling...
Traceback (most recent call last):
  File "/home/mneteler/tmp/grass8-mneteler-2415041/tmpdse8adk2/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/scripts/i.fusion.hpf", line 202, in <module>
    from high_pass_filter import (
ModuleNotFoundError: No module named 'high_pass_filter'
make: *** [/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/include/Make/Html.make:14: i.fusion.hpf.tmp.html] Error 1
ERROR: Compilation failed, sorry. Please check above error messages.
```

**To Reproduce**
Steps to reproduce the behavior:

Tried with `grass83` as well as `grass-dev`.

**Expected behavior**

No python error should occur.

**System description (please complete the following information):**

- Operating System: Fedora 38
- GRASS GIS version:

```
g.version -rge
version=8.3.2dev
date=2023
revision=c003c3f791
build_date=2023-12-15
build_platform=x86_64-pc-linux-gnu
build_off_t_size=8
libgis_revision=493af31836
libgis_date=2023-10-25T07:54:00+00:00
proj=9.1.1
gdal=3.6.4
geos=3.11.1
sqlite=3.40.1
```

as well as today's `grass-dev` ("main").

- Python 3.11.7

Yes:

```
# docker/podman
docker run -it -v .:/src --rm fedora:38 bash
```

First attempt with G8.2.1:
```
# OT: svn still needed as a dependency (if not, I'll update the RPM spec file)?
[root@7f237af87f91 /]# dnf install grass-devel grass make gcc svn

[root@7f237af87f91 /]# grass --tmp-location XY --exec g.extension i.fusion.hpf
Starting GRASS GIS...
Creating new GRASS GIS location <tmploc>...
Cleaning up temporary files...
Executing <g.extension i.fusion.hpf> ...
Fetching <i.fusion.hpf> from GRASS GIS Addons repository (be patient)...
svn: E170013: Unable to connect to a repository at URL 'https://github.com/OSGeo/grass-addons/branches/grass8/src/imagery/i.fusion.hpf'
svn: E160013: '/OSGeo/grass-addons/branches/grass8/src/imagery/i.fusion.hpf' path not found
ERROR: GRASS Addons <i.fusion.hpf> not found
Execution of <g.extension i.fusion.hpf> finished.
Cleaning up temporary files...
```

Second attempt with G8.4:
```
# following https://grasswiki.osgeo.org/wiki/Compile_and_Install#Fedora
[root@5e0cca00834f grass-8.3.git_src_snapshot_2024_01_06]# bin.x86_64-pc-linux-gnu/grass --tmp-location XY --exec g.extension i.fusion.hpf
Fetching <i.fusion.hpf> from <https://github.com/OSGeo/grass-addons/> (be
patient)...
remote: Enumerating objects: 27, done.
remote: Counting objects: 100% (10/10), done.
remote: Compressing objects: 100% (10/10), done.
remote: Total 27 (delta 0), reused 0 (delta 0), pack-reused 17
Receiving objects: 100% (27/27), 1.07 MiB | 8.57 MiB/s, done.
Already on 'grass8'
Your branch is up to date with 'origin/grass8'.
Compiling...
Traceback (most recent call last):
  File "/tmp/grass8-root-23185/tmp36u_6c8k/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/scripts/i.fusion.hpf", line 202, in <module>
    from high_pass_filter import (
ModuleNotFoundError: No module named 'high_pass_filter'
make: *** [/root/grass-8.3.git_src_snapshot_2024_01_06/dist.x86_64-pc-linux-gnu/include/Make/Html.make:14: i.fusion.hpf.tmp.html] Error 1
ERROR: Compilation failed, sorry. Please check above error messages.

```

Here with `--verbose`:

```
GRASS nc_spm_08_grass7/user1:addons > g.extension i.fusion.hpf --verbose
Type of source identified as 'official'.
Fetching <i.fusion.hpf> from <https://github.com/OSGeo/grass-addons/> (be
patient)...
remote: Enumerating objects: 27, done.
remote: Counting objects: 100% (10/10), done.
remote: Compressing objects: 100% (10/10), done.
remote: Total 27 (delta 0), reused 0 (delta 0), pack-reused 17
Receiving objects: 100% (27/27), 1.07 MiB | 3.09 MiB/s, done.
Already on 'grass8'
Your branch is up to date with 'origin/grass8'.
Compiling...
mkdir -p /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/bin
mkdir -p /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/etc
mkdir -p /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/docs/html
mkdir -p /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/docs/man
mkdir -p /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/docs/man/man1
mkdir -p /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/scripts
/usr/bin/install -c i.fusion.hpf.py /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/scripts/i.fusion.hpf
mkdir -p /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/etc/i.fusion.hpf
/usr/bin/install -c -m 644 constants.py /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/etc/i.fusion.hpf/constants.py
/usr/bin/install -c -m 644 high_pass_filter.py /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/etc/i.fusion.hpf/high_pass_filter.py
python3 -t -m py_compile /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/etc/i.fusion.hpf/constants.py
python3 -t -m py_compile /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/etc/i.fusion.hpf/high_pass_filter.py
if [ "/home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/scripts/i.fusion.hpf" != "" ] ; then GISRC=/home/mneteler/tmp/grass8-mneteler-2692454/gisrc GISBASE=/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu PATH="/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/bin:/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/bin:/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/scripts:$PATH" PYTHONPATH="/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/etc/python:/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/gui/wxpython:$PYTHONPATH" LD_LIBRARY_PATH="/home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/bin:/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/bin:/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/scripts:/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/lib:/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/lib:/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/lib" LC_ALL=C LANG=C LANGUAGE=C /home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/scripts/i.fusion.hpf --html-description < /dev/null | grep -v '</body>\|</html>\|</div> <!-- end container -->' > i.fusion.hpf.tmp.html ; fi
Traceback (most recent call last):
  File "/home/mneteler/tmp/grass8-mneteler-2692454/tmpln8xq089/i.fusion.hpf/grass_addons/src/imagery/i.fusion.hpf/scripts/i.fusion.hpf", line 202, in <module>
    from high_pass_filter import (
ModuleNotFoundError: No module named 'high_pass_filter'
make: *** [/home/mneteler/software/grass_main/dist.x86_64-pc-linux-gnu/include/Make/Html.make:14: i.fusion.hpf.tmp.html] Error 1
rm i.fusion.hpf.tmp.html
ERROR: Compilation failed, sorry. Please check above error messages.
```

--- a/src/imagery/i.fusion.hpf/i.fusion.hpf.py
+++ b/src/imagery/i.fusion.hpf/i.fusion.hpf.py
@@ -189,14 +189,7 @@ if "GISBASE" not in os.environ:
 # TODO: avoid PyGRASS altogether
 import grass.script as grass
 
-# from grass.pygrass.raster.abstract import Info
-from grass.pygrass.utils import get_lib_path
-
-# add "etc" directory to $PATH
-path = get_lib_path("i.fusion.hpf", "")
-if path is None:
-    raise ImportError("Not able to find the path %s directory." % path)
-sys.path.append(path)
+grass.utils.set_path(modulename="i.fusion.hpf")
 
 # import modules from "etc"
 from high_pass_filter import (

--- a/src/imagery/i.fusion.hpf/i.fusion.hpf.py
+++ b/src/imagery/i.fusion.hpf/i.fusion.hpf.py
@@ -189,7 +189,7 @@ if "GISBASE" not in os.environ:
 # TODO: avoid PyGRASS altogether
 import grass.script as grass
 
-grass.utils.set_path(modulename="i.fusion.hpf")
+grass.utils.set_path(modulename="i.fusion.hpf", dirname="etc")
 
 # import modules from "etc"
 from high_pass_filter import (

